src/inverseRobot.js

```diff
@@ -23,8 +23,19 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = { ...robot };
```

Comment:
Зачем добавлять в newRobot поля робота, а потом их удалять?
```suggestion
  const newRobot = {};
```